src/schedule.jl

```diff
@@ -13,6 +17,50 @@ mutable struct Schedule
     end
 end
 
+function get_penalties(schedule)::Dict{String,Any}
+    weights = CONFIG["weight_map"]
+    custom_priority = get(schedule.data, "penalty_priorities", nothing)
+    penalties = Dict{String,Any}()
+
+    priority = if !isnothing(custom_priority)
+        custom_priority
+    else
+        CONFIG["penalties"]
+    end
+
+    for (key, pen) in zip(priority, weights)
+        penalties[key] = pen
+    end
+    return penalties
+end
+
+function get_shift_options(schedule::Schedule)
+    get(schedule.data, "shift_types", Dict())
```

Comment:
Trzeba dodać defaultowe.

Reply:
Gadałem na temat tych shiftów we bodajże wtorek z Tomkiem (jak nie było cię na spotkaniu) i doszliśmy do wniosku, że mogą chcieć na frontendzie zmieniać defaultowe zmiany i może lepiej przekazywać wszystko od razu w takim razie. Mogę dodać dla wstecznej kompatybilności, że jak nie ma tego słownika to wczyta defaultowy.

Reply:
Dodać to tutaj to żaden problem, a i tak zawsze request może nadpisać.